Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY ./src/frontend/packages/eslint-config-people/package.json ./packages/eslint
 RUN yarn --frozen-lockfile
 
 ### ---- Front-end builder image ----
-FROM node:20 as frontend-builder
+FROM node:20 as frontend-builder-1
```

Comment:
Just a suggestion: Could we use more descriptive suffixes instead of `-1` and `-2`? It might help make the code clearer.

Reply:
[fixup! 🔨(CI) add Tilt](https://github.com/numerique-gouv/people/pull/178/commits/77bf4271f39fcc03fc1f2bacc0812126772c2e2d)